pkgs/applications/graphics/shotwell/default.nix

```diff
@@ -41,11 +41,11 @@
 
 stdenv.mkDerivation rec {
   pname = "shotwell";
-  version = "0.30.14";
+  version = "0.31.3";
```

Comment:
Looks like we are reverting #141605, shall we update the versionPolicy again?

Reply:
I was running the update script with `GNOME_UPDATE_STABILITY=unstable`. We might just want to drop the commit.